Tests/XCUITests/IntegrationTests.swift

```diff
@@ -68,7 +69,11 @@ class IntegrationTests: BaseTestCase {
     private func waitForInitialSyncComplete() {
         navigator.nowAt(BrowserTab)
         navigator.goto(SettingsScreen)
-        waitForExistence(app.tables.staticTexts["Sync Now"], timeout: 25)
+        waitForExistence(app.tables.staticTexts[ProcessInfo.processInfo.environment["FXA_EMAIL"]!])
```

Comment:
Let's ensure that the settings page is displayed with the account's email first. Sometimes, the settings screen may not be launched successfully.

Reply:
I'm not sure if this is going to work... the email is randomly created and if it is too long, there may appear some ellipsis... so I would remove this line and leave the ones below